### PR TITLE
Add msgs for updating unlocked distribution address and staking rewards distribution address

### PR DESCRIPTION
**Pull request by philipsu522**
## Describe your changes and provide context
Add messages to allow for  updating unlocked distribution address and staking rewards distribution address.  

## Testing performed to validate your change
Unit tests, and also verified on autobake cluster

### CODE REVIEW

File: src/contract.rs [hunk 1]
```diff
@@ -437,6 +449,52 @@ fn execute_propose_update_admin(
...
+fn execute_propose_update_unlocked_distribution_address(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    unlocked_distribution_address: Addr,
+) -> Result<Response<Empty>, ContractError> {
+    let msg = ExecuteMsg::InternalUpdateUnlockedDistributionAddress {
+        unlocked_distribution_address: unlocked_distribution_address.clone(),
+    };
+    let title = format!("updating unlocked distribution address {}", unlocked_distribution_address.to_string());
```
Comment (philipsu522) [inline src/contract.rs:461]: is this title sufficient? not too sure how it's used

File: src/contract.rs [hunk 2]
```diff
@@ -1072,6 +1154,19 @@ mod tests {
                 .add_attribute("proposal_id", 1.to_string())
                 .add_attribute("status", "Open")
         );
+
+        // Verify admin has updated after internal call
```
Comment (philipsu522) [inline src/contract.rs:1158]: add more assertions similar to new ones to ensure that the actual storage values are updated

File: src/contract.rs (hunk 1 above)
Comment (codchen) [reply src/contract.rs:461]: it's purely descriptive

**Review by codchen — APPROVED** [review]
we also need to update the version here https://github.com/sei-protocol/gringotts/blob/main/Cargo.toml#L3

and also the changelog https://github.com/sei-protocol/gringotts/blob/main/CHANGELOG.md

**Comment by philipsu522** [comment]
> we also need to update the version here https://github.com/sei-protocol/gringotts/blob/main/Cargo.toml#L3
> 
> and also the changelog https://github.com/sei-protocol/gringotts/blob/main/CHANGELOG.md

thanks! i actually got an error when testing migration so i bumped it remotely but forgot to make local changes

**Review by besated — APPROVED** [review]

**Pull request merged**